test/dynamo/test_dynamo.py

```diff
@@ -502,7 +502,9 @@ def test_resnet18(self):
     # Graph 3: optimizer
     # Graph 4: sync input for backward
     # Graph 5: sync input for backward (TODO(JackCaoG) understand why there are two graphs)
-    self.assertEqual(met.metric_data('CompileTime')[0], 5)
+    # Graph 6, 7: PyTorch has updated the number of captured by resnet
+    # (https://github.com/pytorch/pytorch/pull/117434)
+    self.assertEqual(met.metric_data('CompileTime')[0], 7)
     # We execute 4 graphs per step when optimizer is enabled.
     self.assertEqual(met.metric_data('ExecuteTime')[0], sample_count * 4)
```

Comment:
Will this number also need to be changed?

Reply:
Yep, it should -- I'm re-running the whole test suite will correct all the values.